framework/src/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -2379,7 +2377,7 @@ public boolean isFromByteCode(Element element) {
      * name equals the passed annotationName if one exists, null otherwise. This
      * is the private implementation of the same-named, public method.
```

Comment:
I see now that you need a version of this method that does not consider aliases.  I would prefer that this method be left private and LockAnnotatedTypeFactory.getDeclAnnotationNoAliases(...) be moved here as it might be generally useful.  

(In general, I think it makes code easier to read if instead of one public method that takes a boolean, two public methods with sensible names call one private method that takes a boolean.)